sample-data/Samples/Sample XML/BearCreek-StudentProgram.xml

```diff
@@ -4808,6 +4856,12 @@
 		<LastEvaluationDate>2021-08-23</LastEvaluationDate>
 		<IEPReviewDate>2021-08-27</IEPReviewDate>
 		<IEPBeginDate>2021-09-01</IEPBeginDate>
+		<Disability>
```

Comment:
Do we need to be providing sample data for OrderOfDisability, particularly in these ones with multiple?  There's some amount of logic in the provided query that uses this property to sort/order the disability records.  It has a coalesce to 1, so it should probably be fine to leave it out, but wanted to bring it up to verify. @vimayya

Reply:
@blmeyers @markramonDL Yes. That would be good considering its use in report query